wazo_calld/plugins/transfers/stasis.py

```diff
@@ -183,34 +196,32 @@ def hangup(self, channel, event):
         self.hangup_pubsub.publish(transfer_role, transfer)
 
     def transfer_recipient_answered(self, channel_event):
+        logger.debug('received TransferRecipientAnswered event: %s', channel_event)
```

Comment:
This is message is misleading, since `TransferRecipientAnswered` is not an event